cranelift/filetests/filetests/isa/s390x/bitops.clif

```diff
@@ -353,11 +353,13 @@ block0(v0: i64):
 ; VCode:
 ; block0:
 ;   flogr %r2, %r2
+;   lgr %r2, %r2
 ;   br %r14
 ; 
 ; Disassembled:
 ; block0: ; offset 0x0
 ;   flogr %r2, %r2
+;   lgr %r2, %r2
```

Comment:
Random drive-by comment, but this looks like something s390x-specific may be required here?